### PR TITLE
Gdaes 392 fixing setup cluster

### PR DESCRIPTION
**Pull request by dwashko**
There has been significant work on the setup cluster functionality of this cookbook. The entire process was rewritten. While it now passes initial testing the setup cluster functionality should be refactored into a class at some point.

### CODE REVIEW

**Comment by akemner** [comment]
looks good

File: metadata.rb [hunk 1]
```diff
@@ -4,13 +4,13 @@
 license          "MIT"
 description      "Installs/Configures Couchbase"
 long_description IO.read(File.join(File.dirname(__FILE__), "README.md"))
-version          "1.0.0"
+version          "1.0.1"
```
Comment (fhanson) [inline metadata.rb]: I would argue that this is more than just a minor version change

**Comment by fhanson** [comment]
Because of the scope of the changes I would bump the cookbook version to 2.0.0, everything else looks good.

**Comment by fhanson** [comment]
Closing to pull into master instead

**Pull request closed without merging**